multiqc/modules/goleft_indexcov/goleft_indexcov.py

```diff
@@ -0,0 +1,144 @@
+"""MultiQC module to plot output from goleft indexcov
+
+https://github.com/brentp/goleft/tree/master/indexcov
+"""
+from __future__ import print_function
+import collections
+import logging
+
+from multiqc import config
+from multiqc.plots import linegraph, scatter
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        super(MultiqcModule, self).__init__(name='goleft indexcov', anchor='goleft_indexcov',
+                                            href='https://github.com/brentp/goleft/tree/master/indexcov',
+                                            info="quickly estimates coverage from a whole-genome bam index.")
+        # Skip coverage plot, too resource intensive with multiple samples
+        # cov_plot = self.coverage_plot()
```

Comment:
If we're dropping support for the coverage plot, do you mind properly deleting the code? We can still go back to it in the version control history and it keeps the codebase cleaner..